koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -560,6 +578,9 @@ def sigint_handler(*args, **kwargs):
         if arch:
             containerdata['arch'] = arch
 
+        if self._user_warnings:
+            containerdata['user_warnings'] = str(self._user_warnings).split('\n')
```

Comment:
:eyes: I closed both my eyes in tests with this usage, but for production code, make a property or something that returns a list

Reply:
Good point